py_gtfs_rt_ingestion/ingest.py

```diff
@@ -110,19 +87,42 @@ def main(event: Dict) -> None:
         config_type = ConfigType.from_filename(files[0])
         converter = get_converter(config_type)
 
-        # filesystem to use when writing parquet files
-        s3_filesystem = fs.S3FileSystem()
-
         for s3_prefix, table in converter.convert(files):
+            # output path is the springboard bucket plus the filetype
             s3_path = os.path.join(export_bucket, s3_prefix)
-            logging.info("Writing Table to %s", s3_path)
-            pq.write_to_dataset(
-                table=table,
-                root_path=s3_path,
-                filesystem=s3_filesystem,
-                partition_cols=converter.partition_cols,
```

Comment:
this function does not work correctly with the `file_visitor` argument.

https://issues.apache.org/jira/browse/ARROW-17068

(they claim its fixed, but its not after looking at source code)

instead, mimic what it was already doing (creating a `Partitioning` object, select defaults for `basename_template`, etc) and call `write_dataset` instead.